src/sui-link-item/index.jsx

```diff
@@ -11,6 +11,7 @@ export default class LinkItem extends Component {
       handleClick: PropTypes.func,
       icon: PropTypes.element,
       literal: PropTypes.string,
+      nofollow: PropTypes.bool,
```

Comment:
Later you can use as here:

```
title={title}
           to={url}
          onClick={handleClick}
          rel={rel}
```

As if it's undefined it won't write anything. Thanks!